factfeel_client.py

```diff
@@ -46,6 +61,12 @@ def set_lights(self, lights):
             if light not in self.lamp_colors:
                 raise NameError(f'Light named {light} does not exist on the Hue Bridge!')
         
+    def xy_to_rgb(self, colors):
+        return [self.converter.xy_to_rgb(x, y) for x, y in colors]
+    
+    def rgb_to_xy(self, colors):
```

Comment:
I don't have a use case for this in mind anyway, but figure it is a nice to have